### PR TITLE
PiJuice performing double conversion

### LINKED ISSUE #1333: PiJuice returns wrong reading in some cases
Created based on @jstoker comment:
https://github.com/dotnet/iot/pull/1213#discussion_r531118940

Per comment GetBatteryCurrent and GetIOCurrent double convert negative numbers resulting in a value 1<<16 less than expected
https://github.com/dotnet/iot/blob/master/src/devices/PiJuice/PiJuiceStatus.cs#L147-L150

cc: @DazFahy

### PR DESCRIPTION
**Pull request by DazFahy**
Fixes #1333

Problem: GetBatteryCurrent and GetIOCurrent perform a double conversion resulting in a value 1<<16 less than expected

Fix: Just perform the BinaryPrimitives.ReadInt16LittleEndian(response) and remove other conversion code,

Tested on my PiJuice for over 1 hour watching the values coming back and all seem to be as expected.

### CODE REVIEW

**Review by Ellerbach — APPROVED** [review]
Thanks for resolving the bug!

File: src/devices/PiJuice/PiJuiceStatus.cs [hunk 1]
```diff
@@ -132,13 +132,7 @@ public ElectricCurrent GetBatteryCurrent()
...
             var response = _piJuice.ReadCommand(PiJuiceCommand.BatteryCurrent, 2);
 
-            int i = BinaryPrimitives.ReadInt16LittleEndian(response);
-            if ((i & (1 << 15)) == (1 << 15))
-            {
-                i -= (1 << 16);
-            }
-
-            return new ElectricCurrent(i, ElectricCurrentUnit.Milliampere);
+            return new ElectricCurrent(BinaryPrimitives.ReadInt16LittleEndian(response), ElectricCurrentUnit.Milliampere);
```
Comment (krwq) [inline src/devices/PiJuice/PiJuiceStatus.cs:135]: this looks good but consider using `ElectricCurrent.FromMilliampere(BinaryPrimitives.ReadInt16LittleEndian(response))`

**Review by krwq — APPROVED** [review]

File: src/devices/PiJuice/PiJuiceStatus.cs (hunk 1 above)
Comment (DazFahy) [reply src/devices/PiJuice/PiJuiceStatus.cs]: Hi @krwq, I've changed all occurrences of UnitsNet constructors to the static From methods in the code.

**Review by krwq — APPROVED** [review]
Thanks for cleaning up remainder

**Pull request merged**